### PR TITLE
Integrations page

### PR DESCRIPTION
**Pull request by ethangj**
CSS changes still need to reflect with cache clear, but staging build here: https://documentation.codeship.com/staging/integrations/integrations/

Let's do code modifications here and structure modifications if any on the Trello card: https://trello.com/c/hntggGzk/283-add-integrations-grid-page

### CODE REVIEW

**Comment by mlocher** [comment]
Looks like this for me, is this accurate?

**Comment by ethangj** [comment]
@mlocher 

Difference between local and staging for me:

screen shot 2017-07-13 at 10 41 53 am

screen shot 2017-07-13 at 10 41 56 am

File: google1a85968316265362.html [hunk 1]
```diff
@@ -1 +0,0 @@
-google-site-verification: google1a85968316265362.html
```
Comment (mlocher) [inline google1a85968316265362.html]: Why did we remove that file?

Comment (ethangj) [inline google1a85968316265362.html]: Afaik we only need that file in there once, when Google initially verifies? I didn't think Google required it to permanently live on the site.

Comment (mlocher) [inline google1a85968316265362.html]: Not sure, but I know other sites which regularly check that file. Is there a negative impact if we keep it?

**Comment by ethangj** [comment]
@mlocher Google file back. Staging/prod are rendering relative fonts slightly different sizes than locally, but got it how I want on staging now.

**Pull request merged**